### PR TITLE
 [New Entry] Added SQL SUBSTRING function

### LINKED ISSUE #4960: [New Entry] Subject: SQL SUBSTRING function
### Reason/inspiration (optional)

Reason/inspiration (optional)
We want a new term entry on the `SUBSTRING` function in SQL. The entry should go into a new folder under docs/content/sql/concepts/string-function/terms/substring/substring.md

The entry should include the following:

1. A description of the term
2. ‘Syntax’ section that includes how to use it
3. Example section to show how to use it

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Topic Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [ ] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by sebasmagno007**
### Description

Added a New Entry term for SQL string functions and created the topic SUBSTRING. My motivation is to contibute to Codeacademy docs in the topics that I am learning.

### Issue Solved
Closes #4960  

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/sql/concepts/string-function/terms/substring/substring.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+We will use an SQL server where we use the next table:
+https://raw.githubusercontent.com/Codecademy/docs/main/media/substring-first.png
+
+This is a correct and functional code using `SUBSTRING` for this table:
+SELECT CustomerID, SUBSTRING(CustomerName, 4, 5) AS alias, SUBSTRING(City, 1, 3) FROM Customers;
+https://raw.githubusercontent.com/Codecademy/docs/main/media/substring-second.png
+
+As you can see we selected the CustomerID column, and the CustomerName column but in this one we used the `SUBSTRING` function slicing it since the fourth character and taking a total of five characters from LEFT to RIGHT, notice that blank spaces are included and we named this column “alias”, we used `SUBSTRING` with the City column taking the first three characters of each row, but we did not use the AS function so it does not have a column_name, be aware to always name your column when using the `SUBSTRING`.
+Feel free to practice in this [data set](https://www.w3schools.com/sql/trysqlserver.asp?filename=trysql_func_sqlserver_substring2).
```
Comment (cigar-galaxy82) [inline content/sql/concepts/string-function/terms/substring/substring.md:46]: Please rather than text and images use code to explain this whole part

**Review by cigar-galaxy82 — CHANGES_REQUESTED** [review]

**Comment by cigar-galaxy82** [comment]
Hi @sebasmagno007 thank you for contributing to Codecademy docs, the entry is well written, I have suggested a few changes please modify

**Comment by sebasmagno007** [comment]
The changes have been made.

**Review by cigar-galaxy82 — APPROVED** [review]

**Review by Sriparno08 — APPROVED** [review]
Looks good, @sebasmagno007!

This entry will be merged soon.

**Pull request merged**